src/steps/writing/creation/rootFiles.ts

```diff
@@ -46,7 +46,7 @@ export async function createRootFiles(options: Options) {
 				},
 			}),
 		}),
-		".nvmrc": `20.8.1\n`,
+		".nvmrc": `20.9.0\n`,
```

Comment:
#877 🫠

Reply:
I noticed that the other error was also caused by renovate bot. I could take a look into this, but I guess this would be out of scope of this PR?